packages/core/src/generators/utils/generate-project.ts

```diff
@@ -188,7 +188,7 @@ export async function GenerateProject(
     sourceRoot: `${normalizedOptions.projectRoot}`,
     targets: {
       build: GetBuildExecutorConfiguration(normalizedOptions.projectRoot),
-      serve: GetServeExecutorConfig(),
+      ...(projectType === 'application' && { serve: GetServeExecutorConfig() }),
```

Comment:
Instead of taking advantage of the fact that {...(false)} is equivalent to {}, would you care to swap this to a ternary?

`...(projectType === 'application' ? { serve: GetServeExecutorConfig() } : {})`

Its a bit clearer, and the other feels like something that could be changed in later node / ecma versions (although it probably will not be due to backwards compat.)

Reply:
No problem. I just learned about this syntax recently, and I couldn't decide which I liked better.

Reply:
Yeah, I had to look it up when I was reviewing the code. Generally if thats the case its best to avoid the syntax and be explicit. Definitely a neat little syntax though, I'll have it floating around my head for a few days before I come to a solid stance on if it is worth using.